contrib/README.md

```diff
@@ -0,0 +1,32 @@
+# Content of the contrib folder
+
+## Build and starting a developement environment
+
+`dev.yml` is a playbook that builds and starts a complete dev environment:
```

Comment:
Lately, there are a lot of changes 
can you verify that it aligned with those changes?